content/docs/user-guide/pipelines/running-pipelines.md

```diff
@@ -1,15 +1,17 @@
 # Running pipelines
 
-To run a pipeline, you can use `dvc exp run`. This will run the pipeline and
-save the results as an [experiment](/doc/user-guide/experiment-management):
+To run a pipeline, you can use either `dvc repro` or `dvc exp run`. Either will
+run the pipeline, and `dvc exp run` will save the results as an
+[experiment](/doc/user-guide/experiment-management) (and has other
+[experiment-related features](/doc/user-guide/experiment-management/running-experiments)
+like modifying parameters from the command line):
```

Comment:
This looks unnecessary though, given the direction we are going.

Reply:
We stopped working on it, and there are still questions about whether we need to keep `repro`, so I'm no longer assuming we are trying to drop `repro`.